staff/alex-barba/gamma-template/style/style.sass

```diff
@@ -0,0 +1,564 @@
+/* Fonts */
+
+@import url('https://fonts.googleapis.com/css?family=PT+Sans')
+
+/* General Rules */
+
+html, body, header, main, section, footer, button
+    margin: 0
+    box-sizing: border-box
+    width: 100%
+    font-family: "Helvetica Neue"
+    color: #333333
+
+h1, h2, h3, h4, h5, h6, p, ul  
+    margin: 0
+
+/* Mixins */
+
+@mixin flex-space-bt
```

Comment:
Here you are not passing any parametre to the mixin, so maybe instead of a it, in that cases you could use the inheritance with %name and extends.